ui/css/perf.css

```diff
@@ -576,7 +577,7 @@ li.pagination-next .material-icons::before {
   width: 0;
   height: 0;
   position: absolute;
-  margin-left: 80px;
+  margin-left: 47%;
```

Comment:
Fixed pixel values would be better than percentage ones here, it's easier to reason about what they're doing.

Reply:
I needed to opt for a percentage because the popup could be a different height/width, depending on content.  Sometimes it has text of "Confidence value as calculated by Perfherder alerts.  Note that...."  and sometimes it's just "4 base / 4 new".  With the pixel value, the shorter popup has the arrow way off to the side.  This at least keeps it very close to the center.

Reply:
Oh, the arrow positioning is based on this value? That's frustrating. :(

Reply:
Yeah, the ``angular1-ui-bootstrap4`` package removed the HTML element that was used for the arrow.  Maybe they found it hard and just removed it to make things easier on themselves.  Not sure.  So I had to get all fancy-pants with a CSS ``after`` element to put it back.  

What's the downside of doing it this way?

Reply:
How about we just kill the arrow for now, since a bunch of other elements don't have them anyway. We can always reintroduce them when/at the point that we port the compare view widget to react.